--- Linux-PAM-1.0.4/modules/pam_console/console.perms~	2009-06-24 16:29:36.000000000 -0700
+++ Linux-PAM-1.0.4/modules/pam_console/console.perms	2009-06-24 16:29:36.000000000 -0700
@@ -18,7 +18,7 @@
 # Rather a new file in the console.perms.d directory should be created.
 
 # file classes -- these are regular expressions
-<console>=tty[0-9][0-9]* vc/[0-9][0-9]* :[0-9]+\.[0-9]+ :[0-9]+
+<console>=tty[0-9] tty[0-9][0-9]* vc/[0-9][0-9]* :[0-9]+\.[0-9]+ :[0-9]+
 <xconsole>=:[0-9]+\.[0-9]+ :[0-9]+
 
 # device classes -- see console.perms.d/50-default.perms
